client/src/components/App.jsx

```diff
@@ -35,13 +35,17 @@ export default function App() {
       .then(setStyles);
   }
 
-  function fetchReviews(count = 2) {
+  function fetchReviews() {
     axios.get('/reviews/meta', { params: { product_id: productID } })
       .then((result) => result.data)
       .then((data) => {
         setReviewsMeta(data);
       });
-    axios.get('/reviews', { params: { product_id: productID, sort: 'relevant', count } })
+    axios.get('/reviews', {
+      params: {
+        product_id: productID, sort: 'relevant', page: 1, count: 500,
+      },
+    })
       .then((result) => result.data.results)
```

Comment:
were you planning on moving this down to the component you need it?

Reply:
Yes, will in a later PR!